File: Leetcode/Daily/10-06-2024/prob.cpp
class Solution {
public:
    int heightChecker(vector<int>& h) {
        vector<int>nums(h.begin(),h.end());

        sort(nums.begin(),nums.end());
        int cnt = 0;
        for(int i=0;i<nums.size();i++){
            cnt += (h[i]!=nums[i]);
        }

        return cnt;
    }
};

File: Leetcode/Daily/10-06-2024/prob.go
func heightChecker(h []int) int {
    nums := make([]int, len(h))
    copy(nums,h)
    sort.Ints(nums)
    cnt := 0
    for i:=0;i<len(h);i++{
        if nums[i]!=h[i]{
            cnt++
        }
    }

    return cnt
}